topics/certs.txt

```diff
@@ -0,0 +1 @@
+The "skyux certs" command contains several actions around certificate management. Available actions are clean, clone, install, trust, uninstall, and untrust. https://developer.blackbaud.com/skyux/learn/reference/cli-commands/generate
```

Comment:
Do we have to include a URL? Can we leave it blank or point to just `cli-commands` until the topic exists?

Reply:
I updated the URL.  We should be able to get that page created before this releases.  I'll go head and create the initial PR now.